NUModules/NUModule.j

```diff
@@ -2077,9 +2077,34 @@ NUModuleTabViewModeIcon                = 2;
 }
 
 /*! Apply the advanced given filers.
-*/
-- (void)applyAdvancedFilters:(CPString)aString
-{
+Date filters will return the default date toString in format Mon May 21 2018 10:59:06 GMT-0700 (PDT)
+We need to intercept predicates and translate dates to format: MM/DD/YYYY HH:MIN:SS timezoneoffset
+*/
+- (void)applyAdvancedFilters:(CPPredicate)aPredicate
+{
+    var subpredicates = [aPredicate subpredicates];
+    var isModified = false;
+    var subPredicates = [];
```

Comment:
subpredicates and subPredicates makes it a bit difficult to read

Reply:
sure!